crates/corro-admin/src/lib.rs

```diff
@@ -347,9 +347,20 @@ async fn handle_conn(
                     }
                     send_success(&mut stream).await;
                 }
-                Command::Cluster(ClusterCommand::MembershipStates) => {
+                Command::Cluster(ClusterCommand::MembershipStates { local }) => {
                     info_log(&mut stream, "gathering membership state").await;
 
+                    if local {
```

Comment:
Ok so generally don't dump all peers we how about but only the ones we are connected to directly.